commands/mr/create/mr_create.go

```diff
@@ -482,7 +482,7 @@ func generateMRCompareURL(opts *CreateOpts) (string, error) {
 		// this uses the slash commands to add labels to the description
 		// See https://docs.gitlab.com/ee/user/project/quick_actions.html
 		// See also https://gitlab.com/gitlab-org/gitlab-foss/-/issues/19731#note_32550046
-		description += fmt.Sprintf("\n/label ~%s", strings.Join(opts.Labels, " ~"))
+		description += fmt.Sprintf("\n/label ~\"%s\"", strings.Join(opts.Labels, " ~"))
```

Comment:
You should use `%q` here instead of `\"%s\"`. Also note that there's a `strings.Join(opts.Labels, " ~")` here which is joining the slice.

Here is a suggestion:
```suggestion
		description += "/label"
		for _, label := range opts.Lables {
			description += fmt.Sprintf(" ~%q", label)
		}
```